bin/rules.d/firefly.rules

```diff
@@ -0,0 +1,9 @@
+# open nodes firefly
+# previous work by Cedric
+# KERNEL=="ttyUSB[0-9]*" , SUBSYSTEMS=="usb", ATTRS{idVendor}=="10c4" , ATTRS{idProduct}=="ea60" , ATTRS{product}=="Zolertia Firefly platform" , MODE="0664", GROUP="dialout", SYMLINK+="ttyON_firefly"
```

Comment:
This was taken from http://zolertia.sourceforge.net/wiki/index.php/Mainpage:Apps